source/index.html

```diff
@@ -1,12 +1,208 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html class="page" lang="ru">
   <head>
     <meta charset="utf-8">
-    <title>HTML Academy: Кэт энерджи</title>
+    <meta name="viewport"
+          content="width=device-width, initial-scale=1">
+    <title>Интернет-магазин функционального питания для котов &laquo;Cat Energy&raquo;</title>
   </head>
-  <body>
 
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS, уровень 2</a>».</p>
+  <body class="page__body">
+    <header class="page-header">
+      <a class="page-header__logo logo
+                logo">
+        <img class="logo__img"
+             src="https://via.placeholder.com/191x38"
+             width="191"
+             height="38"
+             alt="Логотип &laquo;Cat Energy&raquo;">
+      </a>
 
+      <nav class="main-nav">
+
+        <button class="main-nav__toggle
+                       hamburger-menu"
+                type="button">
+          <span class="visually-hidden">Открыть меню</span>
+        </button>
+
+        <div class="main-nav__wrapper">
+          <ul class="main-nav__list
+                     site-list">
+
+            <li class="site-list__item
+                       site-list__item_active">
+              <a class="site-list__link">Главная</a>
+            </li>
+
+            <li class="site-list__item">
+              <a class="site-list__link"
+                 href="catalog.html">Каталог продукции</a>
+            </li>
+
+            <li class="site-list__item">
+              <a class="site-list__link"
+                 href="form.html">Подбор программы</a>
+            </li>
+
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <main class="main-page">
+      <h1 class="visually-hidden">Питательные наборы для особенной диеты вашего кота на сервисе &laquo;Cat Energy&raquo;</h1>
+
+      <section class="promo">
+        <h2 class="promo__title
+                   title--size-36">Функциональное питание для котов</h2>
+
+        <p class="promo__slogan
+                  text--uppercase--size_14">Занялся собой? Займись котом!</p>
+        <a href="fomr.html"
+           class="promo__button
+                  button_green
+                  text--uppercase--size_14"></a>
+      </section>
+
+      <section class="product-line">
+        <h2 class="visually-hidden">Популярные серии наших продуктов для особых диет</h2>
+
+        <ul class="product-line__list
+                   site-list">
+
+          <li class="line
+                     site-list__item">
+
+            <h3 class="line__title title--size-24">Похудение</h3>
```

Comment:
А на десктопе, например, размер будет 28. Так title--size-24 лучше не называть модификаторы

Reply:
Тот же пример из документации запутал. Я уже подумала его изменить 
```suggestion
            <h3 class="line__title title--big">Похудение</h3>
```
Единственное, там шрифты есть просто большие размеры, а есть большие все заглавные и поменьше заглавные.  Словарик нужен..